src/Donations/Routes/RegisterDonationRoutes.php

```diff
@@ -0,0 +1,200 @@
+     *
+     * @unreleased
+     */
+    public function registerGetDonations()
+    {
+        register_rest_route(
+            DonationRoute::NAMESPACE,
+            DonationRoute::DONATIONS,
+            [
+                [
+                    'methods' => WP_REST_Server::READABLE,
+                    'callback' => function (WP_REST_Request $request) {
+                        return $this->donationRequestController->getDonations($request);
+                    },
+                    'permission_callback' => function (WP_REST_Request $request) {
+                        return $this->permissionsCheck($request);
+                    },
+                ],
+                'args' => [
+                    'page' => [
+                        'type' => 'integer',
+                        'default' => 1,
+                        'minimum' => 1,
+                    ],
+                    'per_page' => [
+                        'type' => 'integer',
+                        'default' => 30,
+                        'minimum' => 1,
+                        'maximum' => 100,
+                    ],
+                    'sort' => [
+                        'type' => 'string',
+                        'default' => 'id',
+                        'enum' => [
+                            'id',
+                            'createdAt',
+                            'updatedAt',
+                            'status',
+                            'amount',
+                            'feeAmountRecovered',
+                            'donorId',
+                            'firstName',
+                            'lastName',
+                        ],
+                    ],
+                    'direction' => [
+                        'type' => 'string',
+                        'default' => 'DESC',
+                        'enum' => ['ASC', 'DESC'],
+                    ],
+                    'mode' => [
+                        'type' => 'string',
+                        'default' => 'live',
+                        'enum' => ['live', 'test'],
+                    ],
+                    'campaignId' => [
+                        'type' => 'integer',
+                        'default' => 0,
+                    ],
+                    'includeSensitiveData' => [
+                        'type' => 'boolean',
+                        'default' => false,
+                    ],
+                    'anonymousDonations' => [
+                        'type' => 'string',
+                        'default' => 'exclude',
+                        'enum' => [
+                            'exclude',
+                            'include',
+                            'redact',
+                        ],
+                    ],
+                ],
+            ]
+        );
+    }
+
+    /**
+     * @unreleased
+     */
+    public function permissionsCheck(WP_REST_Request $request)
+    {
+        $isAdmin = current_user_can('manage_options');
```

Comment:
@glaubersilva @jonwaldstein I'm not sure if this can be resolved well now, given our roles and capabilities are a bit garbled, but this is very likely too high of a bar. There are GiveWP roles that are not admins that can view the donor and donation lists.

Reply:
Yeah for sure.  GET request to Donations will probably be `current_user_can('view_give_payments')`.  The potential exception would be a a role of `give_donor` who is requesting a donation that belongs to them.